### PR TITLE
Use scaled avatars for website contributors.

### PR DESCRIPTION
**Pull request by tech4him1**
**- Summary**

Right now we are using full size contributor avatars on the website, which really slows down the loading. This PR allows us to use avatars scaled down to the size that we are using.

**- Test plan**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by verythorough** [comment]
Deploy preview for *netlify-cms-www* ready!

Built with commit 6684a1e910357d7143af74d5e85b8df507169562

https://deploy-preview-1056--netlify-cms-www.netlify.com

**Comment by verythorough** [comment]
Deploy preview for *cms-demo* ready!

Built with commit 6684a1e910357d7143af74d5e85b8df507169562

https://deploy-preview-1056--cms-demo.netlify.com

File: website/gulpfile.babel.js [hunk 1]
```diff
@@ -99,8 +100,17 @@ gulp.task("copy", () =>
         "utf8",
         content =>
           new Promise((resolve, reject) => {
-            const contributors = JSON.parse(content);
-            resolve(yaml.dump({ contributors: contributors.contributors }));
+            const contributors = JSON.parse(content).contributors.map(c => {
+              const avatar = url.parse(c['avatar_url'], true);
+              avatar.search = undefined; // Override so that we can use `avatar.query`.
+              if (avatar.hostname.endsWith('githubusercontent.com')) {
```
Comment (tech4him1) [inline website/gulpfile.babel.js]: Do we even need to check this? I'm guessing the avatars are always going to be from GitHub, and even if not, a query parameter shouldn't break anything.

Comment (erquhart) [reply website/gulpfile.babel.js]: Horse a piece, I'm fine with leaving it. It's explicit at least.

**Pull request merged**